src/QsCompiler/Core/Dependencies.fs

```diff
@@ -41,6 +41,54 @@ type BuiltIn =
     static member RewriteStepDependencies =
         ImmutableHashSet.Create(BuiltIn.RangeReverse.FullName, BuiltIn.Length.FullName)
 
+    /// The set of all built in callables and attributes
```

Comment:
We had chatted offline about the desire for a less ad-hoc setup for rewrite step dependencies at some point.
I am just mentioning it here for the sake of anybody else who wasn't in the conversation. For now, this is ok as is.